eventful_django/eventful_tasks.py

```diff
@@ -13,10 +13,12 @@
 from google.cloud import pubsub_v1
 
 LOGGER = logging.getLogger(__name__)
+environ.pop("CELERY_BROKER_URL", None)
+environ.pop("CELERY_RESULT_BACKEND", None)
```

Comment:
why remove them?

Reply:
Celery reads this env  and neglect eventful envs, so we need to remove these envs to avoid confilcts